server/py/services/api/runtime_handlers/mpijob/abstract.py

```diff
@@ -75,7 +75,7 @@ def run(
             else:
                 pods_phases = self.get_pods(meta.name, meta.namespace)
                 txt = f"MpiJob status unknown or failed, check pods: {pods_phases}"
-                logger.warning(txt)
+                logger.warning(txt, resp_status=mlrun.utils.get_in(resp, "status"))
```

Comment:
```suggestion
                logger.warning("MpiJob status is unknown or failed", pods_phases=pods_phases, resp_status=mlrun.utils.get_in(resp, "status"))
```

a small hassle on my end